var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -13,22 +13,11 @@ class PyNeurodamus(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-py"
 
     version('develop', branch='master')
-    version('2.5.2',   tag='2.5.2')
+    version('2.5.3',   tag='2.5.3')
     version('2.5.0',   tag='2.5.0')
     version('2.4.0',   tag='2.4.0')
     version('2.3.1',   tag='2.3.1')
-    version('2.3.0',   tag='2.3.0')
-    version('2.2.1',   tag='2.2.1')
-    version('2.1.2',   tag='2.1.2')
-    version('2.1.1',   tag='2.1.1')
-    version('2.0.2',   tag='2.0.2')
-    version('2.0.0',   tag='2.0.0')
     version('1.3.2',   tag='1.3.2')
-    version('1.3.1',   tag='1.3.1')
-    version('1.3.0',   tag='1.3.0')
-    version('1.2.1',   tag='1.2.1')
-    version('1.2.0',   tag='1.2.0')
-    version('1.1.0',   tag='1.1.0')
```

Comment:
typically we keep older versions but wouldn't accumulate 30 different versions that no one going to install in practice.

Reply:
Seems reasonable to keep the latest 1.x release and purge the rest. IIRC, at some point we should be able to use git references with Spack. Then having a few earlier versions for dependencies makes sense, and no need to keep 10s of others.

Reply:
So you agree with the current set? I still kept like 10 versions which have a chance of being built, I think all others we can forget (and if ever needed we can checkout spack in another revision)

Reply:
Yes. Thinning out a little at this point is good.